templates/pallet/Cargo.templ

```diff
@@ -14,17 +14,17 @@ codec = { package = "parity-scale-codec", version = "3.0.0", features = ["derive
 scale-info = { version = "2.10.0", default-features = false, features = ["derive"] }
 
 # Substrate
-frame-benchmarking = { version = "26.0.0", default-features = false, optional = true}
-frame-support = { version = "26.0.0", default-features = false}
-frame-system = { version = "26.0.0", default-features = false}
+frame-benchmarking = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.1", default-features = false }
```

Comment:
I added this line to make it work for Sub0, but is there another way?

Otherwise we will have to always update it here when updating base-parachain: https://github.com/r0gue-io/base-parachain/blob/main/Cargo.toml

Reply:
This entire template needs to be updated to use `release-polkadot-*` dependencies. Cannot have the base-parachain using them and this one using `crates-io` dependencies. Issue please.

Reply:
On second thought, I think you've made the appropriate changes already.